src/wallet/test/fuzz/coincontrol.cpp

```diff
@@ -77,6 +77,12 @@ FUZZ_TARGET(coincontrol, .init = initialize_coincontrol)
             },
             [&] {
                 (void)coin_control.GetInputWeight(out_point);
+            },
+            [&] {
+                (void)coin_control.GetSequence(out_point);
+            },
+            [&] {
+                (void)coin_control.GetScripts(out_point);
```

Comment:
These were indeed missing from https://maflcko.github.io/b-c-cov/fuzz.coverage/src/wallet/coincontrol.cpp.gcov.html - there are quite a few more, consider adding those as well

Reply:
Please resolve the comment